Jenkinsfile

```diff
@@ -316,6 +317,51 @@ pipeline {
             }
           }
         }
+        stage('aarch64') {
+          when {
+            beforeAgent true
+            expression { params.aarch64 }
+          }
+          agent {
+            node {
+              label 'aarch64'
+            }
+          }
+          steps {
+            withGithubStatus('aarch64') {
```

Comment:
we've recently removed usage of `withGithubStatus`, so it can be dropped here as well.